R/coord-funs.R

```diff
@@ -304,3 +305,45 @@ coord_pol_centroids <- function(x, lat = NULL, lon = NULL, drop = TRUE) {
 #   coordinates(wc) <- ~long + lat
 #   wc
 # }
+
+
+
+#' @export
+#' @rdname coords
+coord_uncertain<-function(x,coorduncertainityLimit=30000,drop=T,ignore.na=F){
+
+  ## IF variable "coordinateUncertaintyInMeters" doesnt exist
+  if(!("coordinateUncertaintyInMeters" %in% names(x))){
+    stop(" 'coordinateuncertainityInMeters' variable is missing", call. = FALSE)
+  }
+
+#   if(ignore.na==T){
+#     x<-x[!is.na(x$coordinateUncertaintyInMeters),]
+#     uncertain_indices<-which(x$coordinateUncertaintyInMeters > coorduncertainityLimit)
+#     uncertain<-x[uncertain_indices,]
+#   }
+#   else if(ignore.na==F){
+#     NAs<-which(is.na(x$coordinateUncertaintyInMeters))
+#     uncertain_indices<-which(x$coordinateUncertaintyInMeters > coorduncertainityLimit)
+#     uncertain_indices<-c(uncertain_indices,NAs)
+#     uncertain<-x[uncertain_indices,]
+#   }
+
+  if(ignore.na) x<-x[!is.na(x$coordinateUncertaintyInMeters),]
+
+  uncertain_indices<-which(x$coordinateUncertaintyInMeters > coorduncertainityLimit)
+     uncertain<-x[uncertain_indices,]
+
+  if (NROW(uncertain) == 0) uncertain <- NA
+
+  if(drop){
+
+    certain_indices<-setdiff(1:nrow(x),uncertain_indices)
```

Comment:
please have spacing consistent for operators like `=`, `==`, ``<-` , e.g, `certain_indices <- setdiff` instead of `certain_indices<-setdiff`, and spaces after commas